### ISSUE TITLE
Add proper <title>

### ISSUE THREAD

**Issue by seanredmond** [seanredmond/Tindallgrams#24]
The `<title>` element in the `<head>` is empty in the default template. Web pages have titles that describe topic or purpose. Fails [WCAG 2.4.2](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#navigation-mechanisms-title)

**Comment by seanredmond** [comment]
Duplicate of #19
